src/components/campaign/CampaignForm.jsx

```diff
@@ -71,19 +72,27 @@ export default class CampaignForm extends React.Component {
                     let booked = !!userActionList.get('items').find(item =>
                         item.get('id') == action.get('id'));
 
+                    const classes = cx("CampaignForm-action", ( booked ? 'booked' : '' ) );
```

Comment:
For conditional actions like these, use the notation from the `classnames` library, like we did in `ActionForm`. In this case it would be:

```javascript
let classes = cx('CampaignForm-action', {
  booked: booked,
});
```

Ternary operators are difficult to read and should only be used when no better option exists. Also, try to be consistent with single and double quotes.